.github/workflows/gh-pages.yaml

```diff
@@ -0,0 +1,41 @@
+name: CI/CD
+
+on: push
+
+env:
+  # Creates and uploads sourcemaps to Rollbar, and save the built extension as an artifact
+  PUBLIC_RELEASE: ${{ github.ref == 'refs/heads/main' }}
+
+jobs:
+  build-stage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Build react-example
+        working-directory: react-example
+        run: yarn && yarn run build
+
+      - name: Create public distribution
+        # Copy the directory without the directory name, see: https://unix.stackexchange.com/a/180987
+        run: |
+          mkdir public
+          mkdir public/example && cp --recursive example/. public/example
+          mkdir public/react-example && cp --recursive react-example/build/. public/react-example
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: playground-site
+          path: public
```

Comment:
What for?

Reply:
Makes it easier to debug any path issues in projects. Any reason not to include it?

Reply:
> Makes it easier to debug any path issues in projects

The gh-pages deployment pushes to the repo itself:

https://github.com/pixiebrix/playground/tree/gh-pages/react-example

It also appears to preserve the history: https://github.com/pixiebrix/playground/commits/gh-pages

> Any reason not to include it

If the same data is available on the repo, this basically only slows down the deployment (more and more over time).